components/badge.css

```diff
@@ -0,0 +1,58 @@
+.badge {
+    text-align: center;
+    padding: 0.3rem;
+    font-size: 0.9rem;
+    border-radius: 0.7rem;
```

Comment:
Using rem unit for less than 1rem size is not recommended.

Reply:
Use px unit instead.

Reply:
Ok, thanks for suggestion